content/how-to-use-brotli-compression-method/index.md

```diff
@@ -0,0 +1,175 @@
+### Introduction
+
+Data compression is the process of encoding files to reduce their size, making them take up less space on your server and allowing them to be transferred to site clients more quickly. Compression reduces the amount of bandwith required for website rendering, which speeds up the process. Static assets such as *.js*, *.css* and *.json* text files are commonly used in web application projects, with redundancy and compression playing an important role.
+
+### Prerequisites
+
+- Compression algorithms(Lossy & Lossless data compression)
+
+### What is compression algoritm
+
+Compression is the process of reducing the size of files in order to make them more portable.
+
+- Lossless compression restores and rebuilds file data in its original form, after the file is decompressed. For example, when a picture's file size is compressed, its quality remains the same.
+- In Lossy compression, the data in a file is removed and not restored to its original form after decompression. In other words, data is permanently removed.
+During the decompression process, it's also utilized to restore data to its original state.

```

Comment:
More depth explanation is needed. Since your article primarily deals with compression technique